axum-extra/src/routing/typed.rs

```diff
@@ -117,6 +117,28 @@ use super::sealed::Sealed;
 /// struct UsersMember(u32);
 /// ```
 ///
+/// ## Percent encoding
+///
+/// The generated [`Display`] implementation will automatically percent encoding parameters:
```

Comment:
```suggestion
/// The generated [`Display`] implementation will automatically percent-encode parameters:
```

Reply:
Also should it be 'arguments' (or 'parameter values') instead of 'parameters'?

Reply:
Yes thats true!

Reply:
The way I remember it from my CS was "actual arguments" (both start with 'a') and "actual" meaning the actual values and then "format parameters" which are just the names without the values.